build-essential.yaml

```diff
@@ -0,0 +1,15 @@
+package:
+  name: build-essential
+  url: https://packages.debian.org/sid/build-essential
+  version: 12.12
+  epoch: 0
+  description: "virtual compat package for build-base"
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - build-base
```

Comment:
Is there value in having `build-base` `provides=build-essential`?

Reply:
Or, if we expect this to diverge from `build-base`, I think we should just explicitly list the constituent packages, instead of having two layers of indirection when someone wants to see what packages are in `build-essential`.

Reply:
@imjasonh currently elastic builds are unable to build `build-base: provides: build-essential` at all =) github actions can.

Reply:
Also in Debian/Ubuntu build-essential implies coreutils bash dash-binsh so not sure if we should honor that or not.

Reply:
I'd like to. If we're going to have the package it might as well match and I don't think I'm the only ones whose added coretuils to get more feature full commands during builds.

Reply:
decisions; should `coreutils bash-binsh gnu-awk gnu-find` simply be what build-base depends on? cause that would make things just work.